src/main/java/org/mockito/CheckReturnValue.java

```diff
@@ -0,0 +1,23 @@
+package org.mockito;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+/**
+ * Findbugs handles any annotation with name "CheckReturnValue" in return value check.
```

Comment:
Since this annotation is on our core public API methods, can you add a paragraph describing what this annotation means for the end users, written in a way that even a newbie would understand :)

Reply:
Actually, I think we should make this package-protected. It's not intended to be used by Mockito users

Reply:
If you make it package-protected we won't be able to use this annotation on our public types that reside is subpackages.